bindings/gcp/bucket/metadata.yaml

```diff
@@ -0,0 +1,43 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: bindings
+name: gcp.bucket
+version: v1
+status: alpha
+title: "GCP Storage Bucket"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-bindings/gcpbucket/
+binding:
+  output: true
+  operations:
+    - name: create
+      description: "Create an item."
+capabilities: []
+builtinAuthenticationProfiles:
+  - name: "gcp"
+metadata:
+  - name: bucket
+    required: true
+    description: |
+      The bucket name.
+    example: '"mybucket"'
+    type: string
+  - name: decodeBase64
+    type: bool
+    required: false
+    default: 'false'
+    description: |
+      Configuration to decode base64 file content before saving to bucket storage. 
+      (In case of saving a file with binary content). true is the only allowed positive value.
+      Other positive variations like "True", "1" are not acceptable. Defaults to false.
```

Comment:
That "True" is not accepted is not correct. This accepts "1", "True", "yes", etc.

Other components use this:

> Configuration to encode base64 file content before returning the content. (In case of opening a file with binary content).

See bindings/aws/s3.